src/badge/README.md

```diff
@@ -0,0 +1,43 @@
+# Badge
+
+[View prototype on CodePen][codepen].
+
+[codepen]: https://codepen.io/thoughtbot/pen/ZEzROvy
+
+## Usage
+
+### Default
+
+```html
+<span class="tbds-badge">Badge Label</span>
+```
+
+### Pill Badge
+
+```html
+<span class="tbds-badge tbds-badge--pill">Pill Badge</span>
+```
+
+### Hollow Badge
+
+```html
+<span class="tbds-badge tbds-badge--hollow">Hollow Badge</span>
+```
+
+### Link Badge
+
+```html
+<a href="#" class="tbds-badge">Link Badge</a>
+```
+
+### Button Badge
+
+```html
+<button href="#" class="tbds-badge">Button Badge</button>
+```
+
+### Badge with text-level formatting
```

Comment:
Thoughts on moving this example, as well as the `<button>` and `<a>` examples above, under the `### Default` heading, since they are not distinct types of Badges, per se, but just the default Badge used on different elements?

Reply:
Done.